src/app/hierarchical-grid/hierarchical-grid-excel-style-filtering-style/hierarchical-grid-excel-style-filtering-style.component.scss

```diff
@@ -0,0 +1,271 @@
+//     )
+// );
+
+// $custom-list-schema: extend($_light-list,
+//     (
+//         background:(
+//             igx-color: ("secondary", 400)
+//         ),
+//         item-background:(
+//             igx-color: ("secondary", 400)
+//         ),
+//         item-background-hover:(
+//             igx-color: ('secondary', 600)
+//         )
+//     )
+// );
+
+// $custom-checkbox-schema: extend($_light-checkbox,
+//     (
+//         empty-color:(
+//             igx-color: ("primary", 400)
+//         ),
+//         fill-color:(
+//             igx-color: ("primary", 400)
+//         ),
+//         tick-color:(
+//             igx-color: ("secondary", 400)
+//         ),
+//         label-color:(
+//             igx-color: ("primary", 400)
+//         )
+//     )
+// );
+
+// $custom-drop-down-schema: extend($_light-drop-down,
+//     (
+//         background-color:(
+//             igx-color: ("secondary", 400)
+//         ),
+//         item-text-color:(
+//             igx-color: ("primary", 400)
+//         ),
+//         hover-item-background:(
+//             igx-color: ("primary", 400)
+//         ),
+//         hover-item-text-color:(
+//             igx-color: ("secondary", 400)
+//         )
+//     )
+// );
+
+// $custom-light-schema: extend($light-schema,(
+//     igx-grid: $custom-grid-schema,
+//     igx-button: $custom-button-schema,
+//     igx-input-group: $custom-input-group-schema,
+//     igx-checkbox: $custom-checkbox-schema,
+//     igx-drop-down: $custom-drop-down-schema,
+//     igx-list: $custom-list-schema
+// ));
+
+// $custom-grid: igx-grid-theme(
+//     $palette: $dark-palette,
+//     $schema: $custom-light-schema
+// );
+
+// $custom-button: igx-button-theme(
+//     $palette: $dark-palette,
+//     $schema: $custom-light-schema
+// );
+
+// $custom-input-group: igx-input-group-theme(
+//     $palette: $dark-palette,
+//     $schema: $custom-light-schema
+// );
+
+// $custom-list: igx-list-theme(
+//     $palette: $dark-palette,
+//     $schema: $custom-light-schema
+// );
+
+// $custom-checkbox: igx-checkbox-theme(
+//     $palette: $dark-palette,
+//     $schema: $custom-light-schema
+// );
+
+// $custom-drop-down: igx-drop-down-theme(
+//     $palette: $dark-palette,
+//     $schema: $custom-light-schema
+// );
+
+/*INCLUSION by DEFAULT ViewEncapsulation strategy*/
+
+:host {
+    ::ng-deep {
+        @include igx-drop-down($custom-drop-down);
+        @include igx-grid($custom-grid);
+        .igx-excel-filter {
+            @include igx-button($custom-button);
+            @include igx-input-group($custom-input-group);
+            @include igx-checkbox($custom-checkbox);
+            @include igx-list($custom-list);
+            .igx-input-group__input::placeholder {
+                color: #FFCD0F;
+            }
+        }
+    }
+}
+
+.wrapper {
+    margin: 16px;
+}
+
+.photo {
+    vertical-align: middle;
+    max-height: 62px;
+}
+.cell__inner_2 {
+    margin: 1px
+}
+
+.density-chooser {
```

Comment:
This class does not seem to be used in the sample.

Reply:
Addressed.